extensions/community/SetFrameRate.json

```diff
@@ -0,0 +1,73 @@
+{
+  "author": "",
+  "category": "Advanced",
+  "extensionNamespace": "",
+  "fullName": "Set framerate",
+  "helpPath": "",
+  "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAyMy4wLjMsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0iSWNvbnMiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4Ig0KCSB2aWV3Qm94PSIwIDAgMzIgMzIiIHN0eWxlPSJlbmFibGUtYmFja2dyb3VuZDpuZXcgMCAwIDMyIDMyOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+DQo8c3R5bGUgdHlwZT0idGV4dC9jc3MiPg0KCS5zdDB7ZmlsbDpub25lO3N0cm9rZTojMDAwMDAwO3N0cm9rZS13aWR0aDoyO3N0cm9rZS1saW5lY2FwOnJvdW5kO3N0cm9rZS1saW5lam9pbjpyb3VuZDtzdHJva2UtbWl0ZXJsaW1pdDoxMDt9DQo8L3N0eWxlPg0KPHBhdGggY2xhc3M9InN0MCIgZD0iTTI4LjYsMTNjLTEuNiwwLjItMy4xLTAuNS00LTJjLTAuOC0xLjQtMC43LTMuMiwwLjMtNC40Yy0xLjUtMS40LTMuMy0yLjQtNS4yLTNDMTkuMSw1LDE3LjcsNiwxNiw2DQoJcy0zLjEtMS0zLjctMi41Yy0yLDAuNi0zLjgsMS42LTUuMiwzQzgsNy44LDguMiw5LjYsNy4zLDExYy0wLjgsMS40LTIuNCwyLjItNCwyYy0wLjIsMS0wLjQsMi0wLjQsM2MwLDEsMC4xLDIuMSwwLjQsMw0KCWMxLjYtMC4yLDMuMSwwLjUsNCwyYzAuOCwxLjQsMC43LDMuMi0wLjMsNC40YzEuNSwxLjQsMy4zLDIuNCw1LjIsM2MwLjYtMS40LDItMi41LDMuNy0yLjVzMy4xLDEsMy43LDIuNWMyLTAuNiwzLjgtMS42LDUuMi0zDQoJYy0wLjktMS4yLTEuMS0zLTAuMy00LjRjMC44LTEuNCwyLjQtMi4yLDQtMmMwLjItMSwwLjQtMiwwLjQtM0MyOSwxNSwyOC45LDEzLjksMjguNiwxM3oiLz4NCjxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjE2IiBjeT0iMTYiIHI9IjMiLz4NCjwvc3ZnPg0K",
+  "name": "SetFrameRate",
+  "previewIconUrl": "https://asset-resources.gdevelop.io/public-resources/Icons/Line Hero Pack/Master/SVG/UI Essentials/042880726ec32f70c7fb8aeb6db58fdb993344f8eb151c9f7ef67abe5ad25814_UI Essentials_settings_cog_gear.svg",
+  "shortDescription": "Adds an action to change the minimum and maximum framerate.",
+  "version": "0.0.1",
+  "description": "The action to change the framerate allows for choices such as using an in-game menu to let the player pick their own framerate or capping the framerate of a cutscene.",
+  "tags": [
+    "Framerate",
+    "FPS"
+  ],
+  "authorIds": [
+    "FT1FnjHIFfRw0weCrS180EUJHtn2"
+  ],
+  "dependencies": [],
+  "eventsFunctions": [
+    {
+      "description": "Changes the maximum framerate of the game. Set it to 0 for unlimited.",
+      "fullName": "Maximum framerate",
+      "functionType": "Action",
+      "name": "MaxFrameRate",
+      "sentence": "Set the maximum framerate to _PARAM1_",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "runtimeScene._runtimeGame._maxFPS = eventsFunctionContext.getArgument(\"MaximumFrameRate\");",
```

Comment:
Sorry @fizzingpop, we cannot accept extensions that use private class properties, even as a community extension! Reason is that there is no API contract/guarantee for them, thus they may break at any GDevelop update.